install/index.md

```diff
@@ -310,10 +324,31 @@ Windows users can now tap into GPU accelerated data science on their local machi
 
 <i class="fas fa-info-circle text-white"></i> When installing with Docker Desktop, if the container pull command is successful, but the run command hangs indefinitely, [ensure you're on Docker Desktop >= 4.18](https://docs.docker.com/desktop/release-notes/){: target="_blank"}.
 
+<br/>
+<div id="wsl2-sdkm"></div>
+
+### **WSL2 SDK Manager Install**
+[NVIDIA's SDK Manager](https://developer.nvidia.com/sdk-manager){: target="_blank"} gives Windows users a Graphical User Interface (GUI) option to install RAPIDS. It also attempts to fix any environment issues before installing RAPIDS or updating RAPIDS, making it ideal for new WSL users.
+1. Install the [latest NVIDIA Drivers](https://www.nvidia.com/en-us/drivers/){: target="_blank"} on the Windows host.
+2. Download [SDK Manager's Ubuntu version from their website](https://developer.nvidia.com/sdk-manager){: target="_blank"} (requires sign up or login to NVIDIA's Developer community).  Do not install yet.  The rest of the instructions assume that your home directory's `Downloads` folder is where the `.deb` file will be stored.  If this is not the case, please change the directory, as needed, for your system.
+3. Install or update WSL2 and the Ubuntu 22.04 or Ubuntu 24.04 package [using Microsoft's instructions](https://docs.microsoft.com/en-us/windows/wsl/install){: target="_blank"}.  If you are comfortable, you can use Windows PowerShell.  Example: to use PowerShell to install Ubuntu 24.04, type:
+```bash
+wsl --install -d Ubuntu-24.04
+```
+This will install and start Ubuntu in your Windows host system using WSL2.  Make your **sudo** password memorable as you will need it in the next two steps.
+
+4. Install and run SDK Manager inside Ubuntu by pasting this into your command line.  You will have to enter the sudo password you created when you installed Ubuntu and also change folder location to match your home directory in Windows (if you don't know it, open PowerShell, type `$HOME`, and hit enter).
```

Comment:
Instructions like this are difficult for users. We ask them to "paste" but this command is not paste-able because it has placeholders like `[YOUR HOME DIRECTORY ON WINDOWS]`.

Reply:
Also the `$HOME` directory you get from PowerShell isn't the same format as what the user needs for the Ubuntu WSL mount. The user has to recognize how to navigate this discrepancy.


What if we proposed something like:
```bash
cd "$(wslpath -au $(cmd.exe /c 'echo %USERPROFILE%' 2>/dev/null))/Downloads"
```

For me, this navigates directly to `/mnt/c/Users/bdice/Downloads`.